core/server/models/post.js

```diff
@@ -104,7 +104,7 @@ Post = ghostBookshelf.Model.extend({
             }
         });
 
-        this.on('destroyed', function onDestroyed(model) {
+        this.on('destroying', function onDestroying(model) {
```

Comment:
This the fix with the least code, although it would have been possible to retrieve the previous page attributed from the previous method of the model but would have involved to change the emitChange method to check if it is a destroy or update.